### PR TITLE
module 11: Removed Heroku deployment and added azure deployment instead

### PR DESCRIPTION
**Pull request by jatin-28**

### CODE REVIEW

**Review by hugh-emerson — CHANGES_REQUESTED** [review]
You are publicly sharing your app's deployment password. This needs to be secured as any other password (and you should treat this password as compromised - you can reset it through the portal)

**Comment by jatin-28** [comment]
> You are publicly sharing your app's deployment password. This needs to be secured as any other password (and you should treat this password as compromised - you can reset it through the portal)

Done!

**Review by hugh-emerson — CHANGES_REQUESTED** [review]
You are now leaking the new password.

Also, the last deployment failed because you wrapped your curl command in quotes.

File: .travis.yml [hunk 1]
```diff
@@ -32,11 +33,6 @@ after_success:
...
-  script:
-    echo "Deploy to Heroku";
-    docker tag $DOCKER_USER/todo-app:production registry.heroku.com/$HEROKU_APP/web;
-    echo $HEROKU_API_KEY | docker login --username=$HEROKU_LOGIN --password-stdin registry.heroku.com;
-    docker push registry.heroku.com/$HEROKU_APP/web;
-    heroku container:release web --app $HEROKU_APP;
+  script: bash deployToAzure.sh
   on:
-    branch: master
+    branch: module_11
```
Comment (hugh-emerson) [inline .travis.yml:38]: Don't forget to update this.

**Review by hugh-emerson — APPROVED** [review]
Looks good! 👍 Merge when you're ready

But I suggest you delete commented out code (both Python and Dockerfile commands). They're safely stored in git if you do want to switch back to Firefox later.

Resubmit it in Aptem and we can approve it there.

**Pull request merged**